rust-runtime/aws-smithy-http-server/src/clone_box_service.rs

```diff
@@ -0,0 +1,58 @@
+use futures_util::future::BoxFuture;
```

Comment:
Missing SPDX licence header and module description for Rust doc. If this comes from axum with modification, you will need to follow what I did in `error.rs` and add the axum license inline.

Reply:
I would also add a description of why this module is here and how it is used in the codebase as it is not the most immediate code we have.